www/gatsby-config.js

```diff
@@ -75,6 +77,64 @@ module.exports = {
         trackingId: 'UA-41298772-1',
       },
     },
+    {
+      resolve: 'gatsby-plugin-feed',
+      options: {
+        query: `
+         {
+          site {
+            siteMetadata {
+              title
+              description
+              siteUrl
+              site_url: siteUrl
+            }
+          }
+        }`,
+        feeds: [
+          {
+            serialize: ({query: {site, allMarkdownRemark}}) => {
+              return allMarkdownRemark.edges.map(edge => {
+                return Object.assign(
+                  {},
+                  {
+                    title: edge.node.frontmatter.title,
+                    description: edge.node.html,
+                    date: edge.node.fields.date,
+                    url: site.siteMetadata.siteUrl + edge.node.fields.slug,
+                    guid: site.siteMetadata.siteUrl + edge.node.fields.slug,
+                  },
+                );
+              });
+            },
+            query: `
+              {
+                  allMarkdownRemark
+                  (limit: 10, 
+                  filter: {id: {regex: "/_posts/"}}, 
+                  sort: {fields: [fields___date], 
+                  order: DESC}) {
+                    edges {
+                      node {
+                        fields {
+                          date(formatString:"MMMM DD, YYYY, h:mm A")
```

Comment:
I don't think this is actually being used for anything. At least, the feed doesn't display a `<date>` attribute or any dates in this format, but rather:

```xml
<pubDate>Tue, 13 Jun 2017 00:00:00 GMT</pubDate>
```

Reply:
It seems that the gatsby-plugin-feed plugin uses that date attribute in order to generate the pubDate field. If I remove it then the pubDate field disappears as well, although I'm not sure why it doesn't respect the format.

Reply:
Gotcha. Okay then. 😄